From b1a39dbcc28ce7673bd5c391c12680736910690a Mon Sep 17 00:00:00 2001
From: Thomas Renninger <trenn@suse.de>
Date: Fri, 19 Sep 2014 15:34:54 +0200
Subject: [PATCH] network: Try to load xennet

This makes dracut boot from NFS in a xen instance.

bnc#896464, bnc#896259

Signed-off-by: Thomas Renninger <trenn@suse.de>
---
 modules.d/90kernel-network-modules/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90kernel-network-modules/module-setup.sh b/modules.d/90kernel-network-modules/module-setup.sh
index 11fbab9..16ed3a7 100755
--- a/modules.d/90kernel-network-modules/module-setup.sh
+++ b/modules.d/90kernel-network-modules/module-setup.sh
@@ -28,7 +28,7 @@ installkernel() {
         =drivers/net/phy \
         =drivers/net/team \
         =drivers/net/ethernet \
-        ecb arc4 bridge stp llc ipv6 bonding 8021q af_packet virtio_net
+        ecb arc4 bridge stp llc ipv6 bonding 8021q af_packet virtio_net xennet
     hostonly="" instmods iscsi_ibft crc32c iscsi_boot_sysfs
 }
 
